lua/kickstart/plugins/debug.lua

```diff
@@ -85,6 +85,12 @@ return {
     dap.listeners.before.event_exited['dapui_config'] = dapui.close
 
     -- Install golang specific config
-    require('dap-go').setup()
+    require('dap-go').setup {
+      delve = {
+        -- On Windows delve must be run attached or it crashes.
+        -- See https://github.com/leoluz/nvim-dap-go/blob/main/README.md#configuring
+        detached = not vim.fn.has 'win32',
```

Comment:
Hmm I think this will always be false
vim.fn.has will return 0 or 1, and in lua `not 0` and `not 1` will both be false
so I think this needs to be:
```
detached = vim.fn.has 'win32' ~= 0,
```
can you please test that?

Reply:
Oh right, Lua does that funny thing with numbers and boolean ops. My bad, will fix.

Reply:
Fixed

Reply:
Thanks!